spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-binder-sample/spring-cloud-gcp-pubsub-stream-functional-binder-sample-sink/src/main/java/com/example/Sink.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import java.util.function.Consumer;
+
+import com.example.model.UserMessage;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * All-in-one Spring Boot app with a Spring Cloud Stream sink.
+ */
+@SpringBootApplication
+public class Sink {
```

Comment:
Maybe `FunctionalSinkApplication` so that the logs will be easier to attribute to this sample app.